molo/core/api/importers.py

```diff
@@ -270,3 +285,122 @@ def copy_site_languages(self):
+            self.section_tags[page.id] = []
+            for section_tag in nested_fields["section_tags"]:
+                self.section_tags[page.id].append(
+                    section_tag["tag"]["id"])
+
+        # nav_tags
+        #  list -> ["tag"]["id"]
+        # -> Need to fetch and create the nav tags
+        #  THEN create the link between page and nav_tag
+        if (("nav_tags" in nested_fields) and
+                nested_fields["nav_tags"]):
+            self.nav_tags[page.id] = []
+            for nav_tag in nested_fields["nav_tags"]:
+                self.nav_tags[page.id].append(
+                    nav_tag["tag"]["id"])
+
+        # reaction_questions
+        #  list -> ["reaction_question"]["id"]
+        # -> Need to fetch and create the reaction questions
+        #  THEN create the link between page and reaction question
+        if (("reaction_questions" in nested_fields) and
+                nested_fields["reaction_questions"]):
+            self.reaction_questions[page.id] = []
+            for reaction_question in nested_fields["reaction_questions"]:
+                self.reaction_questions[page.id].append(
+                    reaction_question["reaction_question"]["id"])
+
+        # recommended_articles
+        #  list -> ["recommended_article"]["id"]
+        # -> Only need to create the relationship
+        if (("recommended_articles" in nested_fields) and
+                nested_fields["recommended_articles"]):
+            self.recommended_articles[page.id] = []
+            for recommended_article in nested_fields["recommended_articles"]:
+                self.recommended_articles[page.id].append(
+                    recommended_article["recommended_article"]["id"])
+
+        # related_sections
+        #  list -> ["section"]["id"]
+        # -> Only need to create the relationship
+        if (("related_sections" in nested_fields) and
+                nested_fields["related_sections"]):
+            self.related_sections[page.id] = []
+            for related_section in nested_fields["related_sections"]:
+                self.related_sections[page.id].append(
+                    related_section["section"]["id"])
+
+        if ("body" in nested_fields) and nested_fields["body"]:
+            page.body = json.dumps(nested_fields["body"])
+
+        if ("tags" in nested_fields) and nested_fields["tags"]:
+            for tag in nested_fields["tags"]:
+                page.tags.add(tag)
+
+        if (("metadata_tags" in nested_fields) and
+                nested_fields["metadata_tags"]):
+            for tag in nested_fields["metadata_tags"]:
+                page.metadata_tags.add(tag)
+
+        if (("social_media_image" in nested_fields) and
+                nested_fields["social_media_image"]):
+            self.attach_image()
+
+        if ("image" in nested_fields) and nested_fields["image"]:
+            self.attach_image()
+
+        # update the state of the page ?
+        page.save()
```

Comment:
@miltontony It doesn't seem to affect the state of the page, but have added it just in case. I'm not sure how to handle edge cases where we have live articles with drafts etc. I've spoken to Jess about articles that have been unpublished and it seems like there are very few articles that fit this use case currently. Happy to chat through this further if you think it's important to add to this PR.